module/cmd/gravity/cmd/root.go

```diff
@@ -59,7 +59,7 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 				return err
 			}
 
-			gravityAppTemplate, gravityAppConfig := initAppConfig();
+			gravityAppTemplate, gravityAppConfig := initAppConfig()
```

Comment:
hmm, looks like my IDE removed the ; automatically but is more of a stylistic choice? feel free to add back or leave

Reply:
it's not a huge deal either way.